File: BLL/ERPModelsView/BasicData_VM/CurrencyMain_VM.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BLL.ERPModelsView.BasicData_VM
{
   public class CurrencyMain_VM
    {
        public CurrencyMaster_VM CurrencyMaster_VM { get; set; }
        public List<CurrencyRates_VM> CurrencyRates_VM { get; set; }
    }
}

File: Angular_UIL/ClientApp/src/app/models/BasicData/CurrencyMain_VM.ts
import { CurrencyMaster_VM } from "./CurrencyMaster_VM";
import { CurrencyRates_VM } from "./CurrencyRates_VM";

export class CurrencyMain_VM {

  constructor
    (
    public CurrencyMaster_VM: CurrencyMaster_VM,
    public CurrencyRates_VM: CurrencyRates_VM[],
    public UpdateType?: string

    ) { }
}
